build.js

```diff
@@ -0,0 +1,87 @@
+const mkdirp = require('mkdirp')
+const ncp = require('ncp').ncp
+const rimraf = require('rimraf')
+const webpackJs = require('./webpack.js')
+
+var EMAIL_TPL = './extensions/enterprise/pro/emails/templates'
+var BOTPRESS_EDITION = process.env.BOTPRESS_EDITION
+
+const cleanOldBuild = () => {
+  return new Promise((resolve, reject) => {
+    console.log('Cleaning old build')
+    // call: rm -rf lib/
+    rimraf('./lib', err => {
```

Comment:
Have a look at [Promisify](http://bluebirdjs.com/docs/api/promise.promisify.html) and [fromCallback](http://bluebirdjs.com/docs/api/promise.fromcallback.html) which would drastically cut down this file :)